### ISSUE TITLE
[Issue] catch event if client switch/close the server while is connected

### ISSUE THREAD

**Issue by Revyn112** [saltminede/saltychat-docs#9]
It would be nice, to catch an event/command, if the player switch to another server or close the connection, but still has teamspeak open.

Also the client crashes, because the error leeds to flooding the clientlog.

```
2020-04-11 21:16:09.028194|DEBUG   |Salty Chat    |12 |{"Command":1,"ServerUniqueIdentifier":"***************************","Parameter":null}
2020-04-11 21:16:09.028194|DEBUG   |Salty Chat    |13 |Sent 88 bytes
2020-04-11 21:16:09.030196|DEBUG   |Salty Chat    |13 |92 bytes read
2020-04-11 21:16:09.030196|DEBUG   |Salty Chat    |6  |{"Command":2,"ServerUniqueIdentifier":"***************************","Parameter":null}
2020-04-11 21:16:09.405150|INFO    |ClientUI      |1  |Connect to server: otherserver.de
2020-04-11 21:16:09.433176|INFO    |ClientUI      |1  |Connect status: Disconnected
2020-04-11 21:16:09.442185|INFO    |ClientUI      |1  |Disconnected or forced to leave, want autoreconnect = 0
2020-04-11 21:16:09.605351|INFO    |ClientUI      |1  |Connecting to next server...
2020-04-11 21:16:09.605351|INFO    |ClientUI      |1  |Connect to server: otherserver.de
2020-04-11 21:16:09.606353|INFO    |ClientUI      |1  |Trying to resolve otherserver.de
2020-04-11 21:16:09.649922|DEBUG   |Salty Chat    |13 |4096 bytes read
2020-04-11 21:16:09.650922|DEBUG   |Salty Chat    |13 |4096 bytes read
2020-04-11 21:16:09.650922|DEBUG   |Salty Chat    |13 |4096 bytes read
2020-04-11 21:16:09.650922|DEBUG   |Salty Chat    |13 |4096 bytes read
2020-04-11 21:16:09.650922|DEBUG   |Salty Chat    |13 |483 bytes read
2020-04-11 21:16:09.651922|DEBUG   |Salty Chat    |6  |{"Command":14,"ServerUniqueIdentifier":"***************************","Parameter":{"SelfState":{"Position":{"x":-1.5962491035461426,"y":19.240516662597656,"z":71.09185791015625},"Rotation":160.8258514404297,"VoiceEffect":0},"PlayerStates":[]}}
2020-04-11 21:16:09.657928|ERROR   |Plugin Wrapper|   |NotConnected (1794) at C:\Build\saltmine\salty-chat\SaltyChat\TeamSpeak\Channel.cs:65
```

**Comment by BlackFlash5** [comment]
Thanks for reporting this issue.
Should be fixed in 1.3.1 and there is already a command that notifies you, but it was effected by the bug.

`StateUpdate.IsConnectedToServer` and `StateUpdate.IsReady` should now be `false`, when disconnecting from the TeamSpeak server.

Can you give it a try and confirm, that the issue is resolved?

**Comment by Revyn112** [comment]
Can confirm this is fixed now.
